### ISSUE TITLE
update on blur not working when tabbing or pressing enter

### ISSUE THREAD

**Issue by hefloryd** [angular-ui/ui-grid#5970]
- [X] This is a bug report, not a question on how to use the grid.  
Use [Stack Overflow](http://stackoverflow.com/questions/tagged/angular-ui-grid) or [Gitter](https://gitter.im/angular-ui/ui-grid) for questions.
- [X] You have searched the open issues to see if this bug has been filed before.  We don't want duplicate issues.
- [X] You have reviewed the tutorials and documentation to ensure you are using the grid correctly. [ui-grid.info](http://ui-grid.info/docs/#/tutorial)
- [X] You are using the latest version of the grid. Older versions are not patched or supported.
- [X] You have provided steps to recreate your bug.  A plunkr is even better.

It doesn't seem possible to get update on blur working when leaving a cell using the tab or enter keys. This plunker shows the problem:

http://plnkr.co/edit/8tA4aCuaJCb0TAE4kUiu

The plunker is tutorial 301 with editableCellTemplate added on the address.city and name columns.  The editableCellTemplate is the standard cellEditor with update on blur added.

The plunker also watches the model for changes. The age field has no editableCellTemplate and typing into it shows that the model changes with each keystroke. The other two fields do not update on each keystroke but they also don't update if you leave the cell by tabbing or pressing enter. They do update if you click outside of the cell.

This problem has been reported before in #3181 which was closed. Also see PR #4698.

**Comment by aiguofer** [comment]
Hey, did you try that PR to see if it works for you?

**Comment by hefloryd** [comment]
Yes, it looked promising but I had the same issue @imbalind reported, entered data is lost when you leave the last visible column.

**Comment by orneryd** [comment]
Thanks for contributing! This repository has been transferred, and we're preparing the new 5.x release. We're closing older issues and pull requests as part of the reset. Please join us on Discord to follow the work or continue the conversation: https://discord.gg/Baz4w8ZWN
